### ISSUE TITLE
Include "sync_systems" into "rmt-cli sync"?

### ISSUE THREAD

**Issue by digitaltom** [SUSE/rmt#664]
The feature to sync systems to SCC from RMT is currently 'hidden' in `rmt-cli systems scc-sync`. 
A customer was opening a bug report, under the assumption the systems sync will happen as part of the scheduled sync automatically when enabled in the config.

**Comment by thutterer** [comment]
That sounds like a nice improvement to me. We could then soft-depricate the `systems scc-sync` which in fact sounds a bit off and hard to find. Like, leave it there for a few versions, print a deprication warning and remove it in the next major release.

Do you think we should make the systems syncing as part of `sync` configurable? As they may be users who don't intend to sync their systems info back to SCC.

**Comment by digitaltom** [comment]
I agree this needs to be a config option (if it's not there already).

**Comment by felixsch** [comment]
I close this, since the `systems scc-sync` command is used by the systemd timer to automatically sync systems to SCC. There is also the `sync_systems` option in `/etc/rmt.conf` to disable/enable systems syncing!

**Comment by lagartoflojo** [comment]
> under the assumption the systems sync will happen as part of the scheduled sync automatically when enabled in the config.

Isn't that assumption correct?

**Comment by digitaltom** [comment]
> I read: "some command is a bit hidden, and customer was under the correct assumption the command runs automatically."

Yes, it runs automatically as long as the config is set, and the systemd timers are in place (both is default). 
This ticket is more about why this is hidden in a subcommand, when all user SCC sync calls are included in `rmt-cli sync`.
